### PR TITLE
Update Migrating Encrypted Passwords docs

### PR DESCRIPTION
**Pull request by sajinieKavindya**
$subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-esb-5.x.x.md [hunk 1]
```diff
@@ -1353,11 +1353,72 @@ Follow the instructions given below if you have used a **custom log4j component*
...
+		   | username and password  | The name and password of the database user. |
+
+		   
+		b. Open the `registry.xml` file (stored in the `<WUM_UPDATED_ESB_5.0.0_HOME>/conf` directory) and specify the datasource name.
+
+		   ```xml
+		   <dbConfig name="wso2registry">   
+               <dataSource>jdbc/MY_DATASOURCE_NAME</dataSource>
+           </dbConfig>
+           ```
```
Comment (samgnaniah) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-esb-5.x.x.md:1397]: Does this render correctly when you generate the docs locally?

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md [hunk 2]
```diff
@@ -1012,10 +1012,72 @@ Follow the instructions given below if you have used a **custom log4j component*
...
 
-To migrate the encrypted passwords from EI 6.1.0, you need to first obtain the plain-text passwords. Once you have them, follow the normal procedure of encrypting secrets in the Micro Integrator. See [Encrypt Secrets]({{base_path}}/install-and-setup/setup/mi-setup/security/encrypting_plain_text) for instructions.
+To migrate the encrypted passwords from EI 6.1.0, you need to first re-encrypt (using OAEP) all the encrypted information 
+in the database. Then, you can obtain the plain-text passwords by following the normal procedure of encrypting secrets 
+in the Micro Integrator. See [Encrypting Secrets]({{base_path}}/install-and-setup/setup/mi-setup/security/encrypting_plain_text) 
+for instructions.
 
-In case you need to obtain the plain-text passwords by decrypting the encrypted passwords in the EI 6.1.0,
-you can use the [password decryption tool](https://github.com/wso2-docs/WSO2_EI/tree/master/migration-client).
+!!! Info API-M 4.0.0 uses OAEP for data encryption in addition to the RSA algorithm
```
Comment (samgnaniah) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md:1020]: Does this render correctly when you generate the docs locally?

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md [hunk 3]
```diff
@@ -1012,10 +1012,72 @@ Follow the instructions given below if you have used a **custom log4j component*
...
+		   | username and password  | The name and password of the database user. |
+
+		   
+		b. Open the `registry.xml` file (stored in the `<WUM_UPDATED_EI_6.1.0_HOME>/conf` directory) and specify the datasource name.
+
+		   ```xml
+		   <dbConfig name="wso2registry">   
+               <dataSource>jdbc/MY_DATASOURCE_NAME</dataSource>
+           </dbConfig>
+           ```
```
Comment (samgnaniah) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md:1056]: Please check if the formatting works in the docs when you generate.

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.1.md [hunk 4]
```diff
@@ -1042,10 +1042,72 @@ Follow the instructions given below if you have used a **custom log4j component*
 
 ### Migrating encrypted passwords
```
Comment (samgnaniah) [inline en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.1.md:1043]: Apply comments above to all pages if relevant.

**Review by Nirothipan — APPROVED** [review]

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md (hunk 2 above)
Comment (sajinieKavindya) [reply en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md]: fixed

File: en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md (hunk 3 above)
Comment (sajinieKavindya) [reply en/docs/install-and-setup/upgrading-from-wso2-ei/upgrading-from-ei-6.1.0.md:1058]: checked

**Pull request merged**